alpha-blog/app/controllers/users_controller.rb

```diff
@@ -4,7 +4,19 @@ def new
   end
 
   def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:notice] = "Welcome to Alpha Blog #{@user.username}, you have successfully signed up!"
+      redirect_to articles_path
```

Comment:
`articles_path` ?

Reply:
Yep. after user signup will see all articles created by different users in `/articles`